### PR TITLE
Login

### PR DESCRIPTION
**Pull request by angiecstn**
https://www.pivotaltracker.com/n/projects/2198547 

As a user
In order to see my messages
I would like to sign in to my account

We added the user story for the login, set up the features and their corresponding step definitions.
We tested both the happy and sad path.
Regarding the unit tests since they are the same as for the sign up, nothing was added.

We are learning better how to test on Rails since we followed the steps that we did yesterday and we are making sure to double check and really understand everything that was unclear (eg. the importance of naming the parameters between the feature and the step definitions.

### CODE REVIEW

File: features/login_to_the_app.feature [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+Then I should be on the inbox page
+And I should see the message "Signed in successfully"
+
+Scenario: Sad path user not able to login
+Given I am on the landing page
+When I click on the link "Login"
+And I fill in the "Email" with "jorge@kalle.com"
+And I fill in the "Password" with "x"
+When I click on the link "Log in"
+Then I should see the message "Invalid Email or password."
```
Comment (SophieSjoberg) [inline features/login_to_the_app.feature]: Revisit these scenarios and read them out loud. The when and then flow is a bit unorthodox.

File: features/step_definitions/login_step_definitions_step.rb [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  When("I click on the link {string}") do |button|
+    click_on button
+  end
+  
+  When("I fill in the {string} with {string}") do |field, content|
+    fill_in field, with: content
+  end
+  
+  Then("I should be on the inbox page") do
+
```
Comment (SophieSjoberg) [inline features/step_definitions/login_step_definitions_step.rb]: Whats happening here? It is empty

**Review by SophieSjoberg — COMMENTED** [review]
Left comments

File: features/login_to_the_app.feature (hunk 1 above)
Comment (angiecstn) [reply features/login_to_the_app.feature]: fixed

File: features/step_definitions/login_step_definitions_step.rb (hunk 2 above)
Comment (angiecstn) [reply features/step_definitions/login_step_definitions_step.rb]: got rid of it

**Review by SophieSjoberg — APPROVED** [review]
Good job

**Pull request merged**